modules/installation-osp-configuring-floating-ip.adoc

```diff
@@ -9,6 +9,11 @@
 
 After you install {product-title}, configure {rh-openstack-first} to allow application network traffic.
 
+[NOTE]
+====
+You do not need to perform this procedure if you provided values for `platform.openstack.lbFloatingIP` and `platform.openstack.ingressFloatingIP` in the `install-config.yaml` file during installation. The floating IP addresses are already set.
```

Comment:
This note can be removed if this module is moved out of installation assemblies and into a post-installation task one.

Reply:
Eh. I want to leave it.